### PR TITLE
Add   Local communities group to repository first page

### PR DESCRIPTION
**Pull request by hyzx86**
#### Local communities

[Orchard Core CN 中文讨论组](//shang.qq.com/wpa/qunwpa?idkey=48721591a71ee7586316604a7a4ee99d26fd977c6120370a06585085a5936f62)

### CODE REVIEW

File: src/README.md [hunk 1]
```diff
@@ -14,6 +14,12 @@ It’s important to note the differences between the framework and the CMS. Some
 [![Documentation](https://readthedocs.org/projects/orchardcore/badge/)](https://docs.orchardcore.net/)
 [![Crowdin](https://badges.crowdin.net/orchard-core/localized.svg)](https://crowdin.com/project/orchard-core)
 
+#### Local communities
+
+中文资源
+
+[![Orchard Core CN 中文讨论组](https://pub.idqqimg.com/wpa/images/group.png)](//shang.qq.com/wpa/qunwpa?idkey=48721591a71ee7586316604a7a4ee99d26fd977c6120370a06585085a5936f62)
+
```
Comment (Skrypt) [inline src/README.md:22]: I just did merge this. Please remove. Also irrelevant for this PR based on the title of the PR.

**Review by Skrypt — CHANGES_REQUESTED** [review]

File: src/README.md (hunk 1 above)
Comment (hyzx86) [reply src/README.md]: Removed

It's strange that the latest code I got from the main branch didn't get Update of https://github.com/OrchardCMS/OrchardCore/pull/8314/files

**Review by Skrypt — APPROVED** [review]

**Comment by hishamco** [comment]
Is there a difference from the merged [commit](https://github.com/OrchardCMS/OrchardCore/commit/e80384dfe477fe8000a5438035de768387291398)?

**Comment by hyzx86** [comment]
> Is there a difference from the merged [commit](https://github.com/OrchardCMS/OrchardCore/commit/e80384dfe477fe8000a5438035de768387291398)?

this is repository readme.md  
that is src/readme.md  for doc site home page

**Review by hishamco — APPROVED** [review]

**Pull request merged**